### PR TITLE
KOGITO-1465: Resource not found error when navigating directly to /Pr…

### PR DESCRIPTION
**Pull request by nmirasch**
…ocessInstances
react component managing served endpoints

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@nmirasch I'm getting the following error: Uncaught SyntaxError: Unexpected token '<' then nothing loads. Am I missing something? I downloaded the artifact from Jenkins and also build locally.

**Comment by nmirasch** [comment]
@cristianonicolai I changed this a bit adding the exact path to be rerouted. Now it works better, it was a matter of cleaning caches. Could you take a look to this please? Thanks!

**Comment by cristianonicolai** [comment]
@nmirasch I tried to clean the cache, maybe I missed something?

This current approach works, but I guess will be hard to keep covering all the cases, for instance:

http://localhost:8280/DomainExplorer/Travels
http://localhost:8280/ProcessInstances/867c4400-8f72-487d-a734-7ecc21aced34

Should also work, that's why I'm wondering.

**Comment by nmirasch** [comment]
@cristianonicolai Adding this '*' to the path pattern, cover those mentioned cases, the only thing i to take this into account when new friendly url is added. Do you think this could work?

**Review by cristianonicolai — APPROVED** [review]

File: management-console/src/main/java/org/kie/kogito/mgmt/VertxRouter.java [hunk 1]
```diff
@@ -37,7 +38,13 @@ public void init() {
                 .replace("__DATA_INDEX_ENDPOINT__", "\"" + dataIndexHttpURL + "/graphql\"");
     }
 
-    @Route(path = "/", methods = GET)
+    void setupRouter(@Observes Router router) {
+        router.route("/").handler(ctx -> ctx.response().putHeader("location", "/ProcessInstances/").setStatusCode(302).end());
```
Comment (cristianonicolai) [inline management-console/src/main/java/org/kie/kogito/mgmt/VertxRouter.java:42]: @nmirasch this redirect should be handle in React itself, just wondering if it is necessary.

**Pull request merged**